editor/src/components/editor/store/editor-state.ts

```diff
@@ -315,6 +347,21 @@ export function isGithubUpdating(operations: Array<GithubOperation>): boolean {
   return operations.some((operation) => operation.name === 'updateAgainstBranch')
 }
 
+export function isGithubListingPullRequestsForBranch(
+  operations: Array<GithubOperation>,
+
```

Comment:
```suggestion
```
Strange whitespace here

Reply:
I'm surprised that didn't get obliterated by Prettier.